fun will_throw
{
try { fun will_throw
{
void
try {
! 2[ will_throw()]
}
catch const e {
}return 3
}[ will_throw()]
}
catch const e {
}return
}
var exc = void
try {[ will_throw()... ]
}
catch const e {
 e
}
assert exc
